DelegateFun/Sorter.Tests/SortUtilityTests.cs

```diff
@@ -8,6 +9,50 @@ public class SortUtilityTests
         [TestMethod]
         public void SortUtility_ShouldSortAscending_UsingAnAnonymousMethod()
         {
+            //Organize
+            int[] intArr = { 3, 4, 5, 2, 4, 2, 1};
+            SortUtility sortUtil = new SortUtility();
+
+            //Act
+            intArr = sortUtil.Sort(intArr,  delegate(int first, int second)

```

Comment:
```suggestion
            intArr = Sort(intArr,  delegate(int first, int second)
```

Save some typing if you just import Sorter.SortUtil at the top instead.

Reply:
Not sure I understand this. I'm instantiating a SortUtility object so I have to call sortUtil.Sort() from the object. Any clarification would be great :)

Reply:
You can just put "using Sorter.SortUtil;" at the top if you want, then you haven't got to type SortUtil every time you call Sort. Honestly doesn't make much of a difference, just a small refactoring suggestion.